tools/build.hpp

```diff
@@ -15,92 +14,87 @@ namespace antler {
    struct build_project {
       inline build_project(CLI::App& app) {
          subcommand = app.add_subcommand("build", "Build a project.");
-         subcommand->add_option("-p, path", path, "This is the path to the root of the project.")->required();
-      }
-
-      bool should_repopulate() {
-         auto proj = std::filesystem::path(path) / std::filesystem::path(project::project::manifest_name);
-         auto build = std::filesystem::path(path) / std::filesystem::path("build") / std::filesystem::path(project::cmake::cmake_lists);
-
-         auto last_manifest_time = std::filesystem::last_write_time(proj);
-
-         if (!std::filesystem::exists(build)) {
-            return true;
-         }
-
-         auto last_pop_time      = std::filesystem::last_write_time(build);
-
-         return last_pop_time < last_manifest_time;
+         subcommand->add_option("-p, path", path, "This is the path to the root of the project.")->default_val(".");
+         subcommand->add_option("-j, --jobs", jobs, "This is the path to the root of the project.")->default_val(1);

```

Comment:
wrong description

Reply:
I don't think `1` is a good default value for jobs. An acceptable value might be `0` or the user's `submodule.fetchJobs` setting. It doesn't seem friendly to override the users gitconfig, though.

Reply:
Description corrected in 55f8937.
#28 written to address unfriendly jobs behavior.